clusters/resource_cluster.go

```diff
@@ -234,21 +221,61 @@ func fixInstancePoolChangeIfAny(d *schema.ResourceData, cluster *Cluster) {
 	}
 }
 
+func listClusterConfigChangedKeys(d *schema.ResourceData) []string {
```

Comment:
do we _really_ need to introduce `listClusterConfigChangedKeys` or can we live with `hasClusterConfigChanged`?

This PR touches the most critical part of the provider and i want to reduce any refactoring to minimum. could you please revert the introduction of this function?

Reply:
We can live with `hasClusterConfigChanged`, I would just have to manually calculate the change list!

Reply:
removed